build/build.proj

```diff
@@ -84,7 +85,7 @@
   <Target Name="Test" Condition="'$(RunTests)' == 'true'">
 
     <ItemGroup>
-      <TestAssembly Include="$(TestsDirectory)*Tests.dll" />
+      <TestAssembly Include="$(TestsDirectory)*UnitTests.dll" />
```

Comment:
You need to update the version of xunit runner below at line 95 - that's why the tests are failing.,